spec/order_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'order'
+
+describe Order do
+
+
+  it 'initializes 0 total' do
+    expect(subject.total).to eq(0)
+  end
+
+  it 'initializes an empty order list' do
+    expect(subject.order_list).to be_empty
+  end
+
+  it { is_expected.to respond_to(:select).with(2).argument }
+
+  describe ".select" do
+
+    let(:dish){ double :dish}
```

Comment:
Looks like you are making good attempts at a double here! How could we make sure the double is able to call the thing that the method calls?